File: Resueltos_por_competidor/Juan_Martinez/TheatreSquare.cpp
/*
 * Autor: Juan Martinez
 * Problema: Theatre Square (1A)
 * Juez online: Codeforces
 * Veredicto: Accepted
 * Url: https://codeforces.com/problemset/problem/1/A
 * Difficulty: 1000
 */
#include <bits/stdc++.h>

using namespace std;

#define ll long long

int main()
{
    ll n, m, a, fi, co;
    scanf("%lld %lld %lld", &n, &m, &a);
    double x,y;
    x = m;
    y = n;
    fi = ceil(y/a);
    co = ceil(x/a);
    printf("%lld", (co*fi));
    return 0;
}

File: Resueltos_por_competidor/Santiago_Ortiz/TheatreSquare.java
import java.util.Scanner;

/*
 * Autor: Darkovski7   
 * Problema: Theatre Square
 * Juez Online: Codeforces
 * Veredicto: Accepted 
 * Url: https://codeforces.com/problemset/problem/1/A
 */

public class TheatreSquare {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		long n;
		long m;  //recordar numeros que pueden ser muy grandes para un int
		long a;
		long contador;
		
		n = sc.nextLong();
		m = sc.nextLong();
		a = sc.nextLong();
		
		long x = Math.ceilDiv(n, a);
		long y = Math.ceilDiv(m, a);
		contador = x*y;

		System.out.println(contador);
	}

}
